drivers/nfs.py

```diff
@@ -202,7 +202,11 @@ def scan_exports(target):
         textnode = dom.createTextNode(target)
         subentry.appendChild(textnode)
 
-        (path, access) = val.split()
+        # Access is not always provided by showmount return
+        # If none is provided we need to assume "*"
+        array = val.split()
+        path = array[0]
+        access = array[1] if len(array) == 2 else "*"
```

Comment:
What if for some reason len(array) is 3? In the original code this would probably still work but here it would use "*". I'd use `>= 2`.

Reply:
> The commit message should explain what the issue is (in what situation users can meet it, the cause, the consequence for users) and what you chose to do to fix it. I'd also add a reference to the related github issue since we have one for that.

This is a backport, everything appears in the original PR and the issue is linked as well in the original PR, which is linked in the first message.

Would you rather I duplicated what's written there?

Reply:
This should not happen, but I can make the change indeed

Reply:
```suggestion
        access = array[1] if len(array) >= 2 else "*"
```